flintrock/ec2.py

```diff
@@ -71,6 +71,9 @@ def __init__(
         self.vpc_id = vpc_id
         self.master_instance = master_instance
         self.slave_instances = slave_instances
+        # check private mode
+        ec2 = boto3.resource(service_name='ec2', region_name=self.region)
+        self.__private_net = not ec2.Subnet(self.master_instance.subnet_id).map_public_ip_on_launch
```

Comment:
I don't think a double underscore variable name is appropriate here. Those are for special methods and "class-private" members, which is not the case here.